spec/fixtures/custom_nginx.template

```diff
@@ -1,6 +1,8 @@
 # This is a custom nginx configuration template for Kong specs
 
+> if nginx_user ~= "nobody" and nginx_user ~= "nobody nobody" then
 user ${{NGINX_USER}};
+> end
```

Comment:
imho, it is very important that the - already quite complex - Nginx template stays as simple as possible, with as little Lua logic as possible. I would *consider* implementing this on the Lua side, say in `conf_loader.lua`, and do something like:

```lua
-- consider whitespaces between the 2 parts of the directive
local from, _ , err = ngx.re.find(conf.nginx_user, [[^\s*nobody(?:\s+nobody)?\s*$]])
if err then

end

if from then
  conf.nginx_user = nil
end
```

And in the Nginx template, a simpler:

```
> if nginx_user then
user ${{NGINX_USER}};
> end
```

This way, we provide better matching of this directive when considering whether we should or not (whitespaces), and a simpler Nginx template for users out there who still need to scratch their heads reading it when they need a custom one. Thoughts?